From 5fdfccfedb307a06c18c40aed93e1e0c22f26e74 Mon Sep 17 00:00:00 2001
From: celadon <celadon@intel.com>
Date: Fri, 15 Sep 2023 06:30:25 +0000
Subject: [PATCH] Update firmware folder name

This patch updates fodler name according to
latest driver changes

Tracked-On: OAM-112185
Signed-off-by: Padmashree Mandri <padmashree.mandri@intel.com>
---
 intel/avs/{tgl => adl}/dsp_basefw.bin | Bin
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename intel/avs/{tgl => adl}/dsp_basefw.bin (100%)

diff --git a/intel/avs/tgl/dsp_basefw.bin b/intel/avs/adl/dsp_basefw.bin
similarity index 100%
rename from intel/avs/tgl/dsp_basefw.bin
rename to intel/avs/adl/dsp_basefw.bin
-- 
2.39.2

